src/actions/sources.js

```diff
@@ -220,6 +220,11 @@ export function selectLocation(location: LocationObject) {
 
     dispatch(addTab(source.toJS(), 0));
 
+    if (source.get("loadedState") == "unloaded") {
+      if (!location.line) location.line = 1;
```

Comment:
I'd really love to use line 0 & column 0, but we get this:
![screenshot-20171206102351-914x77](https://user-images.githubusercontent.com/23530054/33663168-21d3da14-da8f-11e7-9127-d98892266499.png)